app/scripts/modules/core/src/help/helpContents.js

```diff
@@ -0,0 +1,552 @@
+'use strict';
```

Comment:
This file was refactored away some time ago. Provider-specific help should go in its own file/module.